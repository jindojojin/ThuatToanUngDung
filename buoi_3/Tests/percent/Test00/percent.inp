5
42 8 35 1 14
70 5 9 9 7
63 5 6 6 20
10 8 62 2 18
90 3 2 1 4
