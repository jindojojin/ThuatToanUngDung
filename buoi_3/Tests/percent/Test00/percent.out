100
100
100
50
100
